addons/medical_engine/functions/fnc_commandChanged.sqf

```diff
@@ -0,0 +1,27 @@
+#include "..\script_component.hpp"
+/*
+ * Author: LinkIsGrim
+ * Intercepts group heal attempts and replaces items
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call ace_medical_engine_fnc_commandChanged
+ *
+ * Public: No
+ */
+params ["_group", "_newCommand"];
+
+if (!local _group) exitWith {};
+if !(_newCommand in ["HEAL", "HEAL SOLDIER", "PATCH SOLDIER", "FIRST AID", "HEAL SELF", "SUPPORT"]) exitWith {};
+
+{
+    _x call EFUNC(common,replaceRegisteredItems);
+    if (!unitReady _x) then {
+        [_x] joinSilent (leader _group)
```

Comment:
I'd like a better way to do this, but couldn't find one.

Reply:
Does this work with subgroups (team colors thing)?

Reply:
Good catch. No, it doesn't, but it's simple enough to fix.

Reply:
Solved